diagnostics/app/model/diagnostics/quizzes/Quizzes.scala

```diff
@@ -0,0 +1,85 @@
+package model.diagnostics.quizzes
+
+import java.text.SimpleDateFormat
+
+import common.ExecutionContexts
+import conf.Configuration
+import org.json4s.native.JsonMethods._
+import play.api.Logger
+import shade.memcached.{Configuration => MemcachedConf, Codec, Memcached}
+import org.joda.time.DateTime
+import org.json4s.{DefaultFormats, Extraction}
+import org.json4s.native.Serialization
+import org.json4s._
+
+import scala.concurrent.Future
+import scala.concurrent.duration.Duration
+
+object Quizzes extends ExecutionContexts {
+
+  implicit val json4sFormats = new DefaultFormats {
+    override def dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'")
+  } ++ org.json4s.ext.JodaTimeSerializers.all
+
+  implicit val memcacheDCodec: Codec[QuizAggregate] = new Codec[QuizAggregate] {
+
+    override def serialize(value: QuizAggregate): Array[Byte] = Serialization.write(value).getBytes("UTF-8")
+
+    override def deserialize(data: Array[Byte]): QuizAggregate = parse(new String(data, "UTF-8")).extract[QuizAggregate]
+
+  }
+
+  lazy val host = Configuration.memcached.host.head
```

Comment:
I was just thinking that as a general principle external developers should be able to run all our code on their machine.